daemon/opensnitchd.service

```diff
@@ -1,15 +1,37 @@
 [Unit]
 Description=Application firewall OpenSnitch
 Documentation=https://github.com/evilsocket/opensnitch/wiki
+Documentation=man:systemd.special
+Documentation=man:systemd.service
+Documentation=man:systemd.exec
+Documentation=man:systemd.unit
+
+DefaultDependencies=no
+Before=network-pre.target shutdown.target
+Wants=network-pre.target
+Conflicts=shutdown.target
+# Don't start when 'no-appfw` is in kernel command-line, to allow booting without it.
+ConditionKernelCommandLine=!no-appfw
 
 [Service]
-Type=simple
-PermissionsStartOnly=true
-ExecStartPre=/bin/mkdir -p /etc/opensnitchd/rules
-ExecStart=/usr/local/bin/opensnitchd -rules-path /etc/opensnitchd/rules
+Type=exec
+ConfigurationDirectory=%N/rules
+ConfigurationDirectoryMode=0700
+
+Environment='custom_cfg=%E/%N/rules'
+# Environment='opts=-debug'
```

Comment:
> @lainedfles There is a benign warning if `$opts` is unset. I'd recommend setting an empty variable as a better practice. Like: `Environment='opts='`

Nice catch, i expected for it to become empty in the expansion so i didn't expect that :+1: